chain/vm/evm.go

```diff
@@ -529,11 +547,9 @@ func (evm *EVM) ModifyAssetProfileTx(sender common.Address, data []byte) error {
 	}
 	// judge data's length
 	if len(newData) > types.MaxMarshalAssetLength {
-		err = fmt.Errorf("the length of data by marshal asset more than max length,len(data) = %d ", len(data))
-		if err != nil {
-			evm.am.RevertToSnapshot(snapshot)
-			return err
-		}
+		log.Errorf("the length of data by marshal asset more than max length,len(data) = %d ", len(data))
```

Comment:
首字母没大写，逗号后面没空格，语法很奇怪，没有打印足够的关键信息
```suggestion
		log.Errorf("The length of marshaling asset data exceed limit, len(data) = %d max = %d", len(data), types.MaxMarshalAssetLength)
```